engines/suppliers/app/models/suppliers/orders_frequency.rb

```diff
@@ -0,0 +1,32 @@
+require 'ice_cube'
+
+module Suppliers
+  class OrdersFrequency < ActiveRecord::Base
+    self.table_name = :orders_frequencies
+
+    FREQUENCY_TYPES = { confirmation: 0, delivery: 1 }.freeze
```

Comment:
🙏  can we extract this into a value object already? Then it's going to be harder like with the roles... Besides, by its name I thought its values where daily, weekly, etc. Never in the world I would think that confirmation or delivery would be related to frequency. We should find a better name. Sounds something more like a stage of an order.